cmd/openshift-tests/provider.go

```diff
@@ -117,7 +121,10 @@ func decodeProvider(provider string, dryRun, discover bool) (*exutilcloud.Cluste
 		return config, nil
 
 	default:
-		var providerInfo struct{ Type string }
+		var providerInfo struct {
+			Type         string
+			Disconnected bool
```

Comment:
I'm debating whether to make you clean up someone else's mess.  We already decode e2e.CloudConfig below, it might be more appropriate to add e2e.CloudConfig here as well:

```
type providerInfo struct {
 Type string
 Disconnected bool
 e2e.CloudConfig `json:",inline"`
}
```

Reply:
Done, I think.  I'm happy to leave the code cleaner than I found it